cirq-core/cirq/vis/heatmap_test.py

```diff
@@ -343,3 +343,88 @@ def test_plot_updates_local_config():
         _, ax = plt.subplots()
         random_heatmap.plot(ax)
         assert ax.get_title() == original_title
+
+
+@pytest.mark.usefixtures('closefigures')
+def test_heatmap_plot_highlighted_qubits():
+    value_map = {
+        (grid_qubit.GridQubit(0, 0),): 0.1,
+        (grid_qubit.GridQubit(0, 1),): 0.2,
+        (grid_qubit.GridQubit(0, 2),): 0.3,
+        (grid_qubit.GridQubit(1, 0),): 0.4,
+    }
+    single_qubit_heatmap = heatmap.Heatmap(value_map)
+
+    qubits = np.array([k[0] for k, _ in value_map.items()])
+    highlighted_qubits_indices = [1, 3]
+    highlighted_qubits = qubits[highlighted_qubits_indices]
+
+    expected_linewidths = [2, 4, 2, 4]
+    expected_edgecolors = np.array(
+        [
+            [0.50196078, 0.50196078, 0.50196078, 1.0],  # grey
+            [1.0, 0.0, 0.0, 1.0],  # red
+            [0.50196078, 0.50196078, 0.50196078, 1.0],  # grey
+            [1.0, 0.0, 0.0, 1.0],  # red
```

Comment:
This looks fragile if matplotlib ever decides to adjust the definition of grey.
Can these be replaced with some suitable matplotlib function calls?

Reply:
Great suggestion, thank you!